data/src/main/java/com/globant/data/service/response/PokemonCardResponse.kt

```diff
@@ -3,8 +3,17 @@ package com.globant.data.service.response
 class PokemonCardResponse(
     var id: String,
     var name: String,
+    var nationalPokedexNumber: Int?,
     var imageUrl: String,
     var types: List<String>?,
     var supertype: String?,
-    var subtype: String?
+    var subtype: String?,
+    var evolvesFrom: String?,
+    var hp: String?,
```

Comment:
sure this is nullable? thought it brought "None" instead

Reply:
it's true, it wasn't necessary, removed nullability, thanks